pkgs/crypto/crypto.go

```diff
@@ -593,3 +594,19 @@ func VerifyReconstructedSignature(sig *bls.Sign, validatorPubKey, msg []byte) er
 	}
 	return nil
 }
+
+func ReadEncryptedRSAKey(privKeyPath, privKeyPassPath string) (*rsa.PrivateKey, error) {
+	keyStorePassword, err := os.ReadFile(privKeyPassPath)
```

Comment:
imo we should not ready files in `crypto.go` read the file outside and pass the function the results. easier to test and use.

Reply:
.